client/import.go

```diff
@@ -208,6 +217,8 @@ func (c *Client) Import(ctx context.Context, reader io.Reader, opts ...ImportOpt
 	}
 
 	for i := range imgs {
+		imgs[i].Labels = make(map[string]string)
+		imgs[i].Labels = iopts.imageLabels
```

Comment:
what about merging with existing one? just thinking that if there is existing one.